Pilot_Plant_Dataset_Doc.ipynb

```diff
@@ -0,0 +1,2231 @@
+{
```

Comment:
<p>Christian, I copy/pasted this section into Word and then back in here to edit it more easily. There were several (20+) instances of extra spaces that I removed in the pasted text below, but did not mark in some way. I put my edits in <strong><em>bold, italicized</em></strong> text and used the strikethrough to indicate text to be removed. </p><p> </p><p><strong>Part 1. Description of <em>Del Mar College Glycol Pilot Plant Dataset</em> </strong></p><p>  </p><p><strong><s>[TO REVIEW]</s></strong><s> &nbsp;</s></p><p>Del Mar College is a workforce development college located in Corpus Christi, Texas. Due to the significant increases in South Texas oil and gas production, Corpus Christi has experienced a boom in transporting energy, crude refining, and liquifying natural gas so that it can <strong><em>be</em></strong> exported by sea. As a result, Corpus Christi is becoming one of the largest exporters of energy. This boom requires an influx of skilled plant operations people and technicians. Adding to this shortage is the retirement of many baby-boom generation operations people who possess a high-level of operations and production knowledge about their facilities, <strong><em>which is</em></strong> not easily transferrable. As a result, industry is trying to fill numerous positions with qualified candidates and is funding Del Mar College to help meet those needs. These companies also send their operations people to Del Mar College for ongoing training to keep their knowledge current. </p><p> </p><p>To help provide qualified operations candidates for these positions, Del Mar College has developed curriculum focusing on real-world, hands-on experience using a glycol pilot plant located on campus. Students learn and gain an understanding in areas such as:</p><p> </p><ul><li>Equipment startup, shutdown, and normal operations</li><li>Plant equipment maintenance</li><li>Plant safety issues</li><li>Analyzing data to troubleshoot plant issues that arise</li></ul><p> </p><p>The pilot plant consists of two units: </p><p> </p><ul><li>Hands On Training Unit #1 (HOT-1), which is comprised of tanks, pumps, and feed units in order to circulate water from the feed tank through product tanks and back to the feed tank. </li><li>Hands On Training Unit #3 (HOT-3), which comprises the distillation column unit; a continuous vacuum distillation process that separates a mixture of glycol and water. The students learn the fundamentals of distillation column operations. </li></ul><p> </p><p>NOTE – At the present time (December 2020), the HOT-1 and the HOT-3 units operate independently. There are plans to run them together in series at a future date, but this data reflects them running standalone from each other. &nbsp;</p><p> </p><p>In 2020, Del Mar College graciously agreed to provide OSIsoft the PI System data collected from their glycol pilot plant for publication on the OSIsoft Academic Hub. This hub is a cloud-based platform that supports data analytics in <s>the</s> university curriculum by providing a data infrastructure to host, aggregate, and analyze data. Students are exposed to real-world industrial data which illustrates some of the same engineering concepts being taught in classrooms and labs. </p><p>In November 2020, OSIsoft gained access to Del Mar's 76 data points through &nbsp;<a href="https://www.osisoft.com/pi-system/pi-cloud" target="_blank">PI Cloud Connect</a>. <strong><em>The Hub dataset begins October 13, 2020 and is kept current, although there may be gaps in the data when Del Mar shuts down the pilot plant due to inactivity or maintenance, or if there are communication lapses for some reason.</em></strong> </p><p>This document describes the <strong><em>Del Mar College Glycol Pilot Plant</em></strong> dataset, <strong><em>the time frame in which the dataset was compiled, the data variables that comprise this dataset, the Asset Framework metadata model which describes the hierarchical structure of the pilot plant data, and data quality</em></strong> (<a href="https://app.reviewnb.com/academic-hub/datasets/pull/1/files/#section_1b" target="_blank">Part 1a</a>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/academic-hub/datasets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
done